notebook/static/notebook/js/outputarea.js

```diff
@@ -701,7 +701,16 @@ define([
         var text_and_math = mathjaxutils.remove_math(markdown);
         var text = text_and_math[0];
         var math = text_and_math[1];
-        marked(text, function (err, html) {
+        // Prevent marked from returning inline styles for table cells
+        var renderer = new marked.Renderer();
+        renderer.tablecell = function (content, flags) {
+          var type = flags.header ? 'th' : 'td';
+          var start_tag = '<' + type + '>';
+          var end_tag = '</' + type + '>\n';
+          return start_tag + content + end_tag;
+        };
+        marked(text, { renderer: renderer }, function (err, html) {
+        // marked(text, function (err, html) {
```

Comment:
Do you want to clear away these commented out lines now that the new code is working?